test/lit.cfg

```diff
@@ -240,6 +240,13 @@ if not lit_config.quiet:
 
 #Start cling with nologo
 config.substitutions.append( ('%cling', config.cling + ' --nologo') )
+if True:
+  # Add a substitution for the builds generated include dir before install
```

Comment:
The comment seems to contradict the commit log - is this against picking up the include of whatever built cling or against the make-installed includes?